.github/workflows/edgetest.yml

```diff
@@ -7,14 +7,14 @@ on:
     - cron: '35 17 * * 5'
 jobs:
   edgetest:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
     name: running edgetest
     steps:
       - uses: actions/checkout@v4
         with:
           ref: develop
       - id: run-edgetest
-        uses: edgetest-dev/run-edgetest-action@v1.5
```

Comment:
This was a bad change (https://github.com/lazyscribe/lazyscribe/pull/145#issuecomment-3140557556). Fixed in #147.